config/clusters/awi-ciroh-2/cluster.yaml

```
@@ -0,0 +1,28 @@
+name: awi-ciroh
+provider: gcp # https://console.cloud.google.com/home/dashboard?&project=ciroh-jupyterhub-423218
+gcp:
+  key: enc-deployer-credentials.secret.json
+  project: ciroh-jupyterhub-423218
+  cluster: awi-ciroh-cluster
+  zone: us-central1
```

```suggestion
  zone: us-central1
  billing:
    paid_by_us: false
```

2i2c are not recovering costs for this project